src/js/components/DataTable/DataTable.js

```diff
@@ -277,9 +289,11 @@ const DataTable = ({
           onSort={sortable || sortProp || onSortProp ? onSort : undefined}
           onToggle={onToggleGroups}
           primaryProperty={primaryProperty}
+          scrollOffset={scrollOffset}
```

Comment:
Does this have to do with Mike's button PR or is this coming from master?

Reply:
This is from master. A PR addressing how overflow scroll bars affect column alignment.

Also, Mike's PR is on master now, so that's where I pulled from.